### ISSUE TITLE
Ability to configure items displayed in menu bar

### ISSUE THREAD

**Issue by pierrepo** [Pelican-Elegant/elegant#139]
Hello,

I would really appreciate if one has the ability to configure the menu bar by choosing the items to display. As an example, the default pelican theme supports such a thing with the variable MENUITEMS defined in pelicanconf.py:

```
MENUITEMS = (
    ('Home', '/'),
    ('About', '/pages/about/'),
    ('Python','/pages/python/'),
    ('Tags', '/tag/'),
)
```

**Comment by meduvigo** [comment]
+1

**Comment by asierrayk** [comment]
Really needed, looking forward for it :)

**Comment by talha131** [comment]
@asierrayk Right, I plan to do it in our second milestone for 2020, along with some other changes for the top menu. Thanks for your patience.

**Comment by thinkhuman** [comment]
Would this enable the removal of 'Categories', 'Tags', and 'Archives' from the menu? Love the theme but absolutely don't want that hardcoded into the menu.

**Comment by talha131** [comment]
Yes. that's the plan.

**Comment by asierrayk** [comment]
Is there any update on this issue? I think that i don't have the proper knowledge to help. Thank you

**Comment by iranzo** [comment]
Added PR for this on #733
